Server.MirForms/SMain.cs

```diff
@@ -558,5 +559,14 @@ private void heroesToolStripMenuItem_Click(object sender, EventArgs e)
 
             form.ShowDialog();
         }
+
+        private void CharacterToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            CharacterInfoForm form = new CharacterInfoForm();
+
+            form.ShowDialog();
+
+            form.Dispose();
```

Comment:
why are you disposing it straight after showing it?

Reply:
Fixed